app/src/main/java/com/sunasterisk/youtubebackground/utils/BasePresenter.kt

```diff
@@ -1,7 +1,7 @@
 package com.sunasterisk.youtubebackground.utils
 
 interface BasePresenter<T> {
-    fun onStart()
+    fun onStart(category: String?)
```

Comment:
tương tự, thêm default
```suggestion
    fun onStart(category: String? = null)
```